ui/jobs/components/sge.ts

```diff
@@ -88,7 +88,12 @@ export class SGEComponent extends BaseComponent {
         }
         break;
       case kAbility.Rhizomata:
-        this.rhizomata.duration = 90;
+        if (this.ffxivVersion < 700)
+          this.psyche.duration = 90;
+        break;
+      case kAbility.Psyche:
+        if (this.ffxivVersion >= 700)
+          this.psyche.duration = 60;
```

Comment:
Not as familiar with the `jobs` module as you no doubt are - but should the first `case` statement here still be for `kAbility.Rhizomata` if this is now a tracker for Psyche?